.github/workflows/application-signals-e2e-test.yml

```diff
@@ -238,3 +238,12 @@ jobs:
     with:
       caller-workflow-name: 'main-build'
       staging-wheel-name: ${{ inputs.staging-wheel-name }}
+  
+  # Stand-Alone ADOT/ADOT Gen AI test on EC2
+  adot-genai:
+    needs: [ upload-main-build ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/python-ec2-genai-test.yml@main
```

Comment:
can we name these tests specifically for Genesis which it is? GenAI is too broad.  :)
`python-ec2-genesis-test.yml`